Earthfile

```diff
@@ -1,14 +1,14 @@
-VERSION 0.6
+VERSION 0.8
 FROM golang:1.22
-ARG DOCKER_REPO=ghcr.io/stackitcloud/yawol/
-ARG BINPATH=/usr/local/bin/
-ARG GOCACHE=/go-cache
+ARG --global DOCKER_REPO=ghcr.io/stackitcloud/yawol/
+ARG --global BINPATH=/usr/local/bin/
+ARG --global GOCACHE=/go-cache
```

Comment:
Can't we just move them to the global scope before the FROM?

Reply:
Hmm, that doesn't seem to be enough:

```
$ earthly +packer
 Init 🚀
————————————————————————————————————————————————————————————————————————————————

           buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)

 Build 🔧
————————————————————————————————————————————————————————————————————————————————

              logbus | Setting organization "" and project ""
             +packer | --> FROM +base
         golang:1.22 | --> Load metadata golang:1.22 linux/arm64
Warning: you are not logged into registry-1.docker.io, you may experience rate-limitting when pulling images
Error: Earthfile:351:4 apply FROM hashicorp/packer:: parse normalized named hashicorp/packer:: invalid reference format
in		github.com/stackitcloud/yawol:os-custom-ca+packer
```